projects/jansson/project.yaml

```diff
@@ -4,4 +4,6 @@ auto_ccs:
   - "cmeister2@gmail.com"
 sanitizers:
   - address
+  - memory:
+      experimental: True
```

Comment:
I think you don't need experimental, I won't expect any false positives. Also, I have tested locally for some time, MSan build works fine.